xarray/tests/test_dask.py

```diff
@@ -703,7 +703,7 @@ def test_values(self):
         assert a.values.tolist() == [1, 2]
         assert not a._in_memory
 
-    def test_from_dask_variable(self):
+    def test_from_daskva_riable(self):
```

Comment:
```suggestion
    def test_from_dask_variable(self):
```

Reply:
oh dear. Thanks, I completely missed that, I seem to have pressed a wrong button while committing.